### ISSUE TITLE
redundantParens shouldn't remove option matching in switch statements

### ISSUE THREAD

**Issue by bofeizhu** [nicklockwood/SwiftFormat#1157]
``` swift
var rating: Int? = 3
switch rating {
case (2...)?:
    doSomething()
default:
    break
}
```
`redundantParens` removes () around "2..."

**Comment by nicklockwood** [comment]
Thanks for reporting this, I'll get it fixes ASAP.

As a workaround, you can disable the `redundantParens` just for that line with:

```swift
switch rating {
// swiftformat:disable:next redundantParens
case (2...)?:
```

**Comment by nicklockwood** [comment]
@bofeizhu fixed in 0.49.7
